.gitignore

```diff
@@ -36,3 +36,4 @@ node_modules
 .vendor
 Goopfile.lock
 venv/
+**/coala-log
```

Comment:
Sometimes there are extra logs created in tests folder, to ignore these too I used the `**` :D